rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -195,7 +199,6 @@ def get_balances(
                         })
                     # Always write detected nfts in the DB to have name and address associated
                     db_data.append((nft.token_identifier, nft.name, None, None, 0, address))  # noqa: E501
-
```

Comment:
don't remove lines like this. This is a good thing to have to separate code blocks. Please don't do this